tests/info.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is `info.py` a good name for this? I know that this comment can devolve into bikeshedding... C++ uses `<version>` for this kind of stuff.

Reply:
It mostly needed to be short and simple and not a common existing package. `version.PY2` is longer, not quite as clear, and doesn't actually contain the pybind11 version, which is what you'd rather expect. I'm not sold on the name, though.

Reply:
I agree that `version.py` is an even worse name. I still don't like `info.py`. Anyway, I'm not going to block the PR on this.

Reply:
`plat.py`?

Reply:
Or maybe `env.py`? `env` is a much more common abbreviation for environment than `plat` is for platform. On top of that, the standard library has `platform.py`.